src/model/GeoW_FunctionsExt.ili

```diff
@@ -16,11 +16,11 @@ MODEL GeoW_FunctionsExt
     !!@ fn.since = "2021-11-15";
     FUNCTION GetArea (Objects: OBJECTS OF ANYCLASS; AreaAttr: TEXT): NUMERIC;
 
-    !!@ fn.description = "Zählt die Löcher in einer Fläche. Für 'Objects' können Objekte oder Geometrien angegeben werden. Für 'AreaAttr' soll der Pfad zur Flächen-Geometrie in INTERLIS 2 Syntax angegeben werden. Falls 'Objects' bereits die Geometrien enthält, soll für 'AreaAttr' 'UNDEFINED' übergeben werden. Wird über eine Beziehung die gleiche Geometrie mehrmals gefunden, werden die Löcher mehrmals gezählt.";
-    !!@ fn.param = "Objects: Ausgangsobjekte oder Geometrien. AreaAttr: Pfad zum Geometrieattribut oder UNDEFINED";
-    !!@ fn.return = "Anzahl Löcher";
+    !!@ fn.description = "Zählt die inneren Ringe in einer Fläche. Für 'Object' kann ein Objekt oder Geometrie angegeben werden. Für 'AreaAttr' soll der Attributname der Flächen-Geometrie. Falls 'Object' bereits die Geometrie enthält, soll für 'AreaAttr' 'UNDEFINED' übergeben werden.";
```

Comment:
REQ:
> Für 'AreaAttr' soll der Attributname der Flächen-Geometrie.

Ist kein ganzer Satz. Ich denke der vorherige Beschrieb _Für 'AreaAttr' soll der Pfad zur Flächen-Geometrie in INTERLIS 2 Syntax angegeben werden._ wäre immer noch in Ordnung.